src/main/java/com/flightstats/hub/dao/aws/ClusterContentService.java

```diff
@@ -143,33 +149,66 @@ public ClusterContentService(
     }
 
     @Override
-    public ContentKey insert(String channelName, Content content) throws Exception {
+    public ContentKey insert(String channelName, Content content) {
+        final Content spokeContent = calculateIndexIfLarge(channelName, content);
```

Comment:
This final isn't necessary even though it's used in the lambda. Java should be able to determine that it's effectively final and compile/run just fine.

That said, this is one example of a case where final in a method would be a reasonable group style decision. It's adding value by indicating it's being used in a lambda. In fact, as I type that sentence, I'm pondering if that's something I should start doing...

Regardless, be consistent, there are other variables in the PR used in lambdas that aren't final.

Reply:
Yep...and I think we've landed on preferring not to use finals in methods.